miniconf.py

```diff
@@ -62,29 +88,39 @@ async def command(self, path, value):
             value: The value to write to the path.
 
         Returns:
-            The received response to the command.
+            (code, msg) tuple as a response to the command. `code` is zero for success and `msg` is
```

Comment:
Unfortunately I can't continue commenting on the discussion because it's "outdated".
But I disagree. You say that returning a dict would leak implementation details. I disagree with that perspective. Instead:
* The implementation detail is not a freedom anymore since you now commit to returning certain data.
* Returning a tuple makes this not self-documenting anymore (while the response itself is self-documenting).
* It also prevents any extension of the response data (like for example returning the actual quantized or clipped values used).

The argument that unpacking is sometimes less code is valid. But it's not so important IMO: In our case here with f-strings, it's actually more code to unpack.